src/meta-schema/meta-schema.ts

```diff
@@ -368,6 +392,16 @@ export function getMetaSchema(model: Model): GraphQLSchema {
         return model.i18n.getEnumValueLocalization(enumValue as EnumValue, getResolutionOrder(resolutionOrder, context));
     }
 
+    function getFlexSearchPrimarySort(type: {}):{field: Field, order: 'ASC' | 'DESC'}[]{
+        const rootEntityType = type as RootEntityType;
+        return rootEntityType.flexSearchIndexConfig.primarySort.map(value => {
+            return {
+                field: rootEntityType.getField(value.field)!,
```

Comment:
this won't work if `field` is actually a path (which it can, right?) I think we should just expose the dot-separated path in the meta API.